popup.js

```diff
@@ -124,13 +124,18 @@ define([
 		},
 
 		/**
-		 * Reposition all the popups due to viewport size change.
+		 * Reposition all the popups. It may need to be called when popup's content changes.
 		 * @private
+		 * @fires module:delite/popup#delite-repositioned
 		 */
-		_repositionAll: function () {
+		_repositionAll: function (measureSize) {
```

Comment:
Should list the measureSize argument in the function description (@param ... or something)